cmd/delete_test.go

```diff
@@ -8,6 +8,61 @@ import (
 	"knative.dev/func/mock"
 )
 
+// TestDelete_Namespace ensures that the namespace provided to the client
+// for use when deleting a function is set
+// 1. The flag /env variable if provided
+// 2. The namespace of the function at path if provided
+// 3. The user's current active namespace
+func TestDelete_Namespace(t *testing.T) {
+	root := fromTempDirectory(t)
```

Comment:
```suggestion
func TestDelete_Namespace(t *testing.T) {
	t.Setenv("KUBECONFIG", "/tmp/non-existent")
	root := fromTempDirectory(t)
```

Reply:
Ignoring current user's KUBECONFIG and clearing args from IDEs are now being addressed in separate PRs